examples/ct/ct_handlers_test.go

```diff
@@ -25,6 +24,7 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/mock"
 	"golang.org/x/net/context"
+	"crypto/sha256"
```

Comment:
Sometimes IntelliJ inserts them in the wrong place and they don't get fixed up. I've moved it back.